### PR TITLE
feat: document indyNamespace for ledger config

### PR DESCRIPTION
**Pull request by morrieinmaas**
Also document changes for https://github.com/hyperledger/aries-framework-javascript/issues/1072

This for v0.3.0 onward

related to [this issue 1072](https://github.com/hyperledger/aries-framework-javascript/issues/1072)

tests fail obviously because this need AFJ>=0.3.0
Signed-off-by: Moriarty <moritz@animo.id>

### CODE REVIEW

**Comment by morrieinmaas** [comment]
related to [this issue 1072](https://github.com/hyperledger/aries-framework-javascript/issues/1072)

File: guides/tutorials/issue-a-credential.md [hunk 1]
```diff
@@ -182,3 +182,12 @@ lead to other people knowing your "password" to your wallet.
   Protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0036-issue-credential/README.md)
 - [Issue Credential V2
   Protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0453-issue-credential-v2/README.md)
+
+### Side notes
+
+As of v0.3.0 you should keep in mind that:
+
+1. When when attempting to register a credential on the ledger that is not in your wallet, AFJ will throw an error (as opposed to returning the credential definition from the ledger in prior versions)
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:190]:
```suggestion
1. When when attempting to register a credential that already exists on the ledger but is not in your wallet, AFJ will throw an error (as opposed to returning the credential definition from the ledger in prior versions)
```

File: guides/tutorials/issue-a-credential.md [hunk 2]
```diff
@@ -182,3 +182,12 @@ lead to other people knowing your "password" to your wallet.
...
+2. Attempting to register a new credential definition that is already in the wallet AFJ will return the stored definition _without_ attempting to register it on the ledger.
```
Comment (TimoGlastra) [inline guides/tutorials/issue-a-credential.md:191]:
```suggestion
2. Attempting to register a new credential definition that is already in the wallet in AFJ will return the stored definition _without_ attempting to register it on the ledger.
```

**Review by TimoGlastra — APPROVED** [review]
Nice 👍

**Pull request merged**